src/components/LectureDetailComponents/Review.js

```diff
@@ -0,0 +1,222 @@
+  margin-bottom: 10px;
+  :last-child {
+    border: none;
+  }
+`;
+
+const ReviewTitleSection = styled.div`
+  display: block;
+  margin: 0 0 4px 0;
+`;
+const ReviewWriterInfo = styled.label`
+  display: inline-felx;
+  justtify-content: flex-start;
+  font-size: 12px;
+  color: ${FontColor};
+`;
+const ReviewStarSection = styled.div`
+  float: right;
+`;
+const StarIcon = styled.img.attrs({
+  src:
+    "https://hangang-storage.s3.ap-northeast-2.amazonaws.com/assets/img/LecturesDetailPage/star.png",
+  alt: "star",
+})`
+  width: 16px;
+  height: 16px;
+`;
+const HalfStarIcon = styled.img.attrs({
+  src:
+    "https://hangang-storage.s3.ap-northeast-2.amazonaws.com/assets/img/LecturesDetailPage/half-star.png",
+  alt: "half-star",
+})`
+  width: 16px;
+  height: 16px;
+`;
+
+const ReviewWrapper = styled.div`
+  height: auto;
+
+  overflow-y: auto;
+  ::-webkit-scrollbar {
+    display: none; /* Chrome, Safari, Opera*/
+  }
+`;
+const ReviewContent = styled.p`
+  margin: 8px 0;
+  font-size: 14px;
+  line-height: 1.4;
+  text-align: left;
+  color: ${FontColor};
+`;
+
+const ContentReportSection = styled.div``;
+const ReportButton = styled.a`
+  float: right;
+  margin: 4px 0 0 0;
+
+  font-size: 12px;
+  text-align: right;
+  color: ${PlaceholderColor};
+  cursor: pointer;
+`;
+
+const ThumbUpSection = styled.div`
+  display: inline-flex;
+  justify-content: flex-start;
+  cursor: pointer;
+`;
+const ThumbUpIcon = styled.img.attrs(({ isLiked }) => ({
+  src: isLiked
+    ? "https://hangang-storage.s3.ap-northeast-2.amazonaws.com/assets/img/resourcepage/thumb_up_pushed.png"
+    : "https://hangang-storage.s3.ap-northeast-2.amazonaws.com/assets/img/resourcepage/thumb_up.png",
+  alt: "thumb up",
+}))`
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+`;
+
+const ThumbUpPushedIcon = styled(ThumbUpIcon).attrs({
+  src:
+    "https://hangang-storage.s3.ap-northeast-2.amazonaws.com/assets/img/resourcepage/thumb_up_pushed.png",
+  alt: "thumb upped ",
+})`
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+`;
+const FilterPickSection = styled.div`
+  margin-left: 4px;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+`;
+const FilterPickLabel = styled.label`
+  margin: 0 2px 0 0;
+  font-size: 12px;
+  font-weight: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+  color: ${FontColor};
+`;
+const LowArrowIcon = styled.img.attrs({
+  src:
+    "https://hangang-storage.s3.ap-northeast-2.amazonaws.com/assets/img/LecturesDetailPage/arrow-low.png",
+  alt: "arrow low",
+})`
+  width: 9px;
+  height: 5px;
+  margin-left: 4px;
+`;
+const SubLabelGrey = styled.label`
+  margin: 4px 4px 8px 0;
+  font-size: 12px;
+  color: ${PlaceholderColor};
+`;
+const SubLabelInfo = styled.label`
+  display: block;
+  margin: 4px 29px 8px 0;
+  font-size: 12px;
+  color: ##222222;
+`;
+const SubLabel = styled.label`
+  margin: 4px 29px 8px 4px;
+  font-size: 12px;
+  color: ##222222;
+`;
```

Comment:
" ## " 뭘까요 ... 글고 #222222는 FontColor import 할 수 있겠네요

```suggestion
const SubLabelInfo = styled.label`
  display: block;
  margin: 4px 29px 8px 0;
  font-size: 12px;
  color: #222222;
`;
const SubLabel = styled.label`
  margin: 4px 29px 8px 4px;
  font-size: 12px;
  color: #222222;
`;
```